index.js

```diff
@@ -1,21 +1,15 @@
 'use strict';
-var applescript = require('applescript');
+const applescript = require('applescript');
+const pify = require('pify');
 
-module.exports = function (app, cb) {
+module.exports = app => {
 	if (process.platform !== 'darwin') {
-		throw new Error('Only OS X systems are supported');
+		return Promise.reject(new Error('Only OS X is supported'));
 	}
 
 	if (typeof app !== 'string') {
-		throw new TypeError('Expected name of an app');
+		return Promise.reject(new TypeError('Expected name of an app'));
 	}
 
-	applescript.execString('quit app "' + app + '"', function (err) {
-		if (err) {
-			cb(err);
-			return;
-		}
-
-		cb();
-	});
+	return pify(applescript.execString)(`quit app "${app}"`);
```

Comment:
What about using [`run-applescript`](https://github.com/sindresorhus/run-applescript) instead and we can get rid of `pify`?